search-ui/src/pages/Details/Details.js

```diff
@@ -34,37 +38,116 @@ export default function Details() {
 
   }, [id]);
 
+  useEffect(() => {
+    setHighlight(null);
+  }, [q]);
+
+  function GetTagsHTML(tags) {
+
+    if (!!tags) {
+      let tagsHtml = tags.map((tagValue, index) => {
+        if (index < 10) {
+
+          if (tagValue.length > 30) { // check tag name length
```

Comment:
Can tagValue ever be null here?

Reply:
It shouldn't be unless our service returns a null value :)